app/scripts/controllers/emissionCtrl.js

```diff
@@ -167,38 +160,44 @@ var emissionCtrl = function($scope, $sce, walletService, libreService, $rootScop
                     getContractData("readyOracles"),
                     getContractData("oracleCount"),
                     getContractData("requestTime"),
-                    walletService.wallet == null ? { data: 0} :
+                    walletService.wallet == null ? { data: 0 } :
                         getTokenData("balanceOf", [walletService.wallet.getAddressString()]),
                     walletService.wallet == null ? { data: 0 } :
                         getTokenData("allowance", [walletService.wallet.getAddressString(), bankAddress])            
                 ]).then((values) => {
                     let 
                         state = values[0],
-                        balance = values[1],
-                        RURCost = values[2],
+                        exchangerTokenBalance = values[1],
+                        updateRatesCost = values[2],
                         calcTime = values[3],
                         readyOracles = values[4],
                         oracleCount = values[5],
                         requestTime = values[6],
-                        tokenBalance = values[7],
+                        userTokenBalance = values[7],
                         allowedTokens = values[8];
 
+                    ajaxReq.getBalance(bankAddress, function(balanceData) {
```

Comment:
It's may load slowly and will be load later. It's ok?

Reply:
yes